### ISSUE TITLE
"Finetune not found" error occurs even after I uploaded LoRA adapters successfully

### ISSUE THREAD

**Issue by takassh** [cloudflare/cloudflare-docs#14757]
### Existing documentation URL(s)

- https://developers.cloudflare.com/workers-ai/fine-tunes/loras/

### tl;dr
Give me more instructions about how to upload Lora adapters.

### description
I created the adapters from `google/gemma-2b-it`.
Then I added `"model_type": "gemma"` to my `adapter_config.json`. (Documentation looks wrong tho. [it](https://developers.cloudflare.com/workers-ai/fine-tunes/loras/#uploading-lora-adapters) says `adapter_model.config`).
And then I uploaded my adapter files with `npx wrangler ai finetune create <model_name> <finetune_name> <folder_path>` and I got `✅ Assets uploaded, finetune "{my_finetune_name}" is ready to use.`.
Finally I called
```
curl https://api.cloudflare.com/client/v4/accounts/{ACCOUNT_ID}/ai/run/@cf/google/gemma-2b-it-lora \
  -H 'Authorization: Bearer {API_TOKEN}' \
  -d '{
    "messages": [{"role": "user", "content": "Hello world"}],
    "raw": "true",
    "lora": "{my_finetune_id}"
  }'
```
but it responds 
```
{
    "errors": [
        {
            "message": "AiError: AiError: Finetune {my_finetune_id} not found",
            "code": 3037
        }
    ],
    "success": false,
    "result": {},
    "messages": []
}
```

So I have no idea about what I am wrong.

### Additional information

Here is my `adapter_config.json`

```
{
  "alpha_pattern": {},
  "auto_mapping": null,
  "base_model_name_or_path": "google/gemma-2b-it",
  "bias": "none",
  "fan_in_fan_out": false,
  "inference_mode": true,
  "init_lora_weights": true,
  "layer_replication": null,
  "layers_pattern": null,
  "layers_to_transform": null,
  "loftq_config": {},
  "lora_alpha": 32,
  "lora_dropout": 0.05,
  "megatron_config": null,
  "megatron_core": "megatron.core",
  "modules_to_save": null,
  "peft_type": "LORA",
  "r": 8,
  "rank_pattern": {},
  "revision": null,
  "target_modules": [
    "model.layers.17.self_attn.o_proj",
    "model.layers.17.self_attn.q_proj",
    "model.layers.17.self_attn.v_proj",
    "model.layers.17.mlp.gate_proj",
    "model.layers.17.mlp.down_proj",
    "model.layers.17.mlp.up_proj",
    "model.layers.17.self_attn.k_proj"
  ],
  "task_type": "CAUSAL_LM",
  "use_dora": false,
  "use_rslora": false,
  "model_type": "gemma"
}
```

**Comment by mchenco** [comment]
hey! we're taking a look - there's a bug we're fixing. i'll ping here to ask you to test again

and good call on the `adapter_model.config` -> `adapter_config.json` - will change that in docs.

**Comment by mchenco** [comment]
@takassh could you try again?

**Comment by takassh** [comment]
@mchenco great! it works! 👍
